### ISSUE TITLE
App is down :(

### ISSUE THREAD

**Issue by JonZavialov** [NateDimick/funholidaysapi#2]

**Comment by NateDimick** [comment]
Yes it is. I'm sorry that you're affected by it. The reason is, as you might be aware, Heroku is [eliminating their free tier](https://blog.heroku.com/next-chapter) at the end of this month. I didn't want to get billed so the app was proactively shut down. 

If I ever get enough free time then I might look in to other free hosting options for the app. Google Cloud Run was appealing but when I tried signing up a few weeks back it was broken.

**Comment by JonZavialov** [comment]
That's alright, it's a shame Heroku is doing that! In the meantime, if anyone else sees this they are welcome to use my endpoint at `https://api.jonzav.me/v1/national-days/?month=<month>&day=<day>`
